packages/patternfly-4/react-core/src/components/Avatar/Avatar.tsx

```diff
@@ -13,8 +13,8 @@ export interface AvatarProps
 }
 
 export const Avatar: React.SFC<AvatarProps> = ({
-  className = '',
```

Comment:
These empty string defaults should be added back!

Reply:
Thanks removed the default props like we discussed and added these back.